### PR TITLE
Improve wording and grammar [3/3]

### PR DESCRIPTION
**Pull request by rubenwardy**

### CODE REVIEW

**Comment by rubenwardy** [comment]
This is blocking progress generally. I wonder whether it should be merged and anything odd fixed afterwards

File: _en/quality/clean_arch.md [hunk 1]
```diff
@@ -167,27 +166,27 @@ to the Minetest API at all.
...
+as shown above, in order to reduce how much you need to change if you modify
 an area's internals or externals. For example, to change the formspec you
 would only need to edit the view. To change the view API, you would only need to
 change the view and the controller, but not the model at all.
 
 In practice, this design is rarely used because of the increased complexity
 and because it doesn't give many benefits for most types of mods. Instead,
-you tend to see a lot more of a less formal and strict kind of design -
-varients of the API-View.
+you tend to see much more of a less formal and strict kind of design -
```
Comment (Ezhh) [inline _en/quality/clean_arch.md]: more/less together like this is ikky.
Maybe "Instead, you will more commonly see a less formal ..." ?

File: _en/quality/clean_arch.md [hunk 2]
```diff
@@ -167,27 +166,27 @@ to the Minetest API at all.
...
+variants of the API-View.
 
 
 ### API-View
 
 In an ideal world, you'd have the above 3 areas perfectly separated with all
 events going into the controller before going back to the normal view. But
-this isn't the real world. A good half-way house is to reduce the mod into 2
+this isn't the real world. A good compromise is to reduce the mod into 2
```
Comment (Ezhh) [inline _en/quality/clean_arch.md]: two, not 2.

File: _en/quality/releasing.md [hunk 3]
```diff
@@ -61,31 +62,31 @@ in any copies of the mod or of substantial parts of the mod.
...
 * How to install the mod.
-* What dependencies there are / what the user needs to install.
-* Where to report problems/bugs or get help.
+* Current version of the mod.
+* Optionally, the where to report problems or get help.
 
 ### description.txt
 
-This should explain what your mod does. Be concise without being vague.
+This should explain what your mod does. Be concise without being too vague.
```
Comment (Ezhh) [inline _en/quality/releasing.md]: I think the original is better because how to measure what "too vague" as opposed to "an okay amount of vague" is?

**Pull request merged**